tests/modules.py

```diff
@@ -5,6 +5,11 @@
 import subprocess
 import os
 import shutil
+import secrets
+from base64 import b64encode
+import cryptography.hazmat.backends as crypto_backends
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import rsa, padding
```

Comment:
Could you move the crypto-specific code to `infra/crypto.py`?

Reply:
Done, I also changed the test endpoint to accept PEM instead of base64-encoded DER, which was silly in the first place.